src/python/dxpy/bindings/dxjob.py

```diff
@@ -38,14 +38,15 @@
 from ..utils.local_exec_utils import queue_entry_point
 from ..compat import basestring
 
+
 #########
 # DXJob #
 #########
 
-def new_dxjob(fn_input, fn_name, name=None, tags=None, properties=None, details=None,
-              instance_type=None, depends_on=None,
-              cluster_spec=None, fpga_driver=None, system_requirements=None, system_requirements_by_executable=None,
-              **kwargs):
+
+def new_dxjob(fn_input, fn_name, name=None, tags=None, properties=None, details=None, instance_type=None,
+              depends_on=None, cluster_spec=None, fpga_driver=None, nvidia_driver=None, system_requirements=None,
```

Comment:
can you please move `nvidia_driver` argument to the end of the arguments list in all cases? adding it in the middle is actually API breaking change and can cause troubles if anyone is using this method without keyword args...

Reply:
sure, fixed.